config/main.go

```diff
@@ -49,6 +50,12 @@ type TwitterPlatformConfig struct {
 	ConsumerSecret    string `json:"consumer_secret"`
 }
 
+type TelegramPlatformConfig struct {
+	ApiID    int    `json:"api_id"`
+	ApiHash  string `json:"api_hash"`
+	BotToken string `json:"bot_token"`
+}
```

Comment:
We can do it, using any public channel where the bot is a member!

Reply:
Wait, you mean a public group (instead of a public channel)?

This is a "channel": everyone can subscribe and read, but only `admin` can post contents

![image](https://user-images.githubusercontent.com/1191636/203824942-72528a91-f04e-4cad-b268-1c665c3fc46a.png)